metapool/metapool.py

```diff
@@ -30,6 +30,8 @@ def read_plate_map_csv(f, sep='\t'):
     ------
     UserWarning
         If there are wells with no sample names associated with them.
+    AssertionError
+        If there are repeated sample names.
```

Comment:
Just to clarify, because I'm not sure given all that I've encountered thus far, should sample names themselves be unique? Or is it the tuple of sample name and lane number that must be unique?